packages/primitives-core/package.json

```diff
@@ -3,14 +3,20 @@
   "version": "9.2.9",
   "description": "Shared utilities for emotion primitives and native",
   "main": "dist/index.cjs.js",
+  "types": "types/index.d.ts",
   "module": "dist/index.esm.js",
   "files": [
     "src",
-    "dist"
+    "dist",
+    "types"
   ],
+  "scripts": {
+    "test:typescript": "dtslint types"
+  },
   "license": "MIT",
   "dependencies": {
     "@emotion/is-prop-valid": "^0.6.3",
+    "@types/react-native": "^0.56.19",
     "css-to-react-native": "^2.2.1"
```

Comment:
Missing deps for `create-emotion` and `create-emotion-styled` where this one uses their type.

Reply:
@mitchellhamilton @nitin42 Adding such deps is acceptable?